mkosi/__init__.py

```diff
@@ -2255,3 +2260,13 @@ def run_verb(args: MkosiArgs, presets: Sequence[MkosiConfig]) -> None:
 
             if args.verb == Verb.serve:
                 run_serve(last)
+
+
+def normalize_mtime(source_date_epoch: Optional[int], root: Path) -> None:
+    if source_date_epoch is None:
+        return
+
+    with complete_step("Setting mtime to SOURCE_DATE_EPOCH"):
```

Comment:
What about "Normalizing modification times of {directory}"

Reply:
Agreed but with a relative path from root to directory

Reply:
Exactly